query-service-impl/src/test/java/org/hypertrace/core/query/service/QueryRequestBuilderUtils.java

```diff
@@ -181,4 +181,70 @@ public static Filter createTimeFilter(String columnName, Operator op, long value
         .setOrder(order)
         .setExpression(createColumnExpression(columnName));
   }
+
+  public static Filter createFilter(
```

Comment:
Do we need this function as this file already contains function - `createTimeFilter`, `createFilter`, `createStringLiteralValueExpression`, `createLongLiteralValueExpression` and `createIntLiteralValueExpression`